File: ejerciciosMariet/Python-ejercicios/ejercicio22.py
num = int(input("Digite el primer número entero: "))
num2 = int(input("Digite el segundo número entero: "))
num3 = int(input("Digite el tercer número entero: "))
num4 = int(input("Digite el cuarto número entero: "))

if num > num2 and num2 > num3 and num3 > num4:
    print("El número mayor es:", num)
elif num < num2 and num2 > num3 and num3 > num4:
    print("El número mayor es:", num2)
elif num < num2 and num2 < num3 and num3 > num4:
    print("El número mayor es:", num3)
elif num2 > num and num > num3 and num3 < num4:
    print("El número mayor es:", num4)

File: ejerciciosMariet/Java-ejercicios/Ejercicio_22.java
import java.util.Scanner;

public class Ejercicio_22 {
    public static void main(String[] args) {
        int num;
        int num2;
        int num3;
        int num4;
       
        
        Scanner scanner = new Scanner(System.in);
        
        System.out.print("Digite el primer número entero: ");
        num = scanner.nextInt();
        System.out.print("Digite el segundo número entero: ");
        num2 = scanner.nextInt();
        System.out.print("Digite el tercero número entero: ");
        num3 = scanner.nextInt();
        System.out.print("Digite el cuarto número entero: ");
        num4 = scanner.nextInt();
        
        if(num > num2 && num2 > num3 && num3 > num4) {
    
            
            System.out.println("El numero mayor es: " + num);

        } else if(num < num2 && num2 > num3 && num3 > num4) {
            
            
            System.out.println("El numero mayor es: " + num2);

        } else if(num < num2 && num2 < num3 && num3 > num4) {
            
            
            System.out.println("El numero mayor es: " + num3);                     
        } else if(num2 > num && num > num3 && num3 < num4) {
           
            System.out.println("El numero mayor es: " + num4);  
}

        
        
        
        
        
        scanner.close();
    }
}
